homework_basic_components/src/task4/Task4.java

```diff
@@ -0,0 +1,63 @@
+package task4;
```

Comment:
You should reconsider refactoring this code in order to reuse it.
There should be two sections, one for handling input and one for handling choices.